tests/home-page-check

```diff
@@ -0,0 +1,9 @@
+import { test } from "@framework/test";
```

Comment:
@lipmaneth playwright test files should have a .js or .ts extension. They are either javascript or typescript code.